### DISCUSSION TITLE
Local Account

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#130] [Q&A] [answered]
Hello,

Loved that you made this! Just a quick question though. Is it possible to run this module on a local account if I don't do the 200IQ for lock screen? 

Thanks!

**Comment by HotCakeX** [comment, marked as answer]
Hi, Thank you,
Sure, you can even skip that category if you want. 200IQ items are totally optional.
________________________________
From: magnetism116 ***@***.***>
Sent: Tuesday, October 3, 2023 8:55 PM
To: HotCakeX/Harden-Windows-Security ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [HotCakeX/Harden-Windows-Security] Local Account (Discussion #130)


Hello,

Loved that you made this! Just a quick question though. Is it possible to run this module on a local account if I don't do the 200IQ for lock screen?

Thanks!

**Reply by ghost** [reply]
thank you!

**Reply by HotCakeX** [reply]
You're welcome!
